src/FileManipulator.php

```diff
@@ -93,13 +93,17 @@ public function performConversions(ConversionCollection $conversions, Media $med
 
                 $copiedOriginalFile = $imageGenerator->convert($copiedOriginalFile, $conversion);
 
-                $conversionResult = $this->performConversion($media, $conversion, $copiedOriginalFile);
+                if (! $conversion->getManipulations()->isEmpty()) {
+                    $manipulationResult = $this->performManipulations($media, $conversion, $copiedOriginalFile);
+                } else {
```

Comment:
Could your refactor this so we don't need an `else` statement? A dedicated method with an early return seems the way to go.